cypress/integration/collections_spec.js

```diff
@@ -106,11 +105,10 @@ describe('Dashboard Collections Page', () => {
     });
 
     it('should display expected MMT Links for a collections list', () => {
-      cy.route({
-        method: 'GET',
-        url: '/collections?limit=*',
-        response: 'fixture:collections-with-mmtLinks.json',
-      }).as('getCollections');
+      cy.intercept(
+        { method: 'GET', url: new RegExp('/collections\\?limit=.*') },
```

Comment:
`cy.intercept` doesn't seem to use string globbing like cy.route did.  So any of the urls with asterisks need to be regexps.

Reply:
That's a great question.  I think there is something different happening in the two cases.  It's not documented, but I see different behavior in testing.

So the documentation says that a string type url input, uses minimatch as the matcher.

`url = "http://localhost:5001/collections?limit=50&page=1&sort_key%5B%5D=-duration&getMMT=true&includeStats=true"
`
`Cypress.minimatch(url, '/collections?limit=*') => false
`
and the `minimatch.makeRe(url) => /^(?:\/(?=.)collections[^/]limit=[^/]*?)$/`

Which doesn't match the url
`url.match(/^(?:\/(?=.)collections[^/]limit=[^/]*?)$/) => null
`

But on extra testing, I could have switched the url to `'**/collections?limit=*'`:
`Cypress.minimatch(url, '**/collections?limit=*') => true
`

So to make sure, I changed L28 to this:
```
cy.intercept({method:'GET', url:'/collections/active?limit=*'}).as('getActiveCollections');
```

And that fails the first test, when it waits for that alias.

TL:TL; 
I think I don't want to change the existing RegExps even though they could be simplified, but I did verify that the intercepts on L26-30 are correctly being intercepted, in the given form, but when a routeMatcher object is provided, those same glob-strings fail.

Reply:
I might post an issue with Cypress, because I don't think I'm misreading the documentation.

Reply:
Posted an issue to Cypress, but it turns out it's probably my problem. When you use a routeMatcher with that style of match, you need to set matchUrlAgainstPath: true like this:
```
      cy.intercept({method:'GET', url:'/collections/active?limit=*', matchUrlAgainstPath: true }).as('getActiveCollections');
```
But I will not go back and change them since a RegExp is also acceptable. 
Issue https://github.com/cypress-io/cypress/issues/14458

Reply:
Cool, thanks for investigating. Seems like it's easier to just use the RegEx than to add an additional param anyway. Good to know there's a way to get the routeM matching working though!